plugins/livelog/livelog.go

```diff
@@ -100,11 +104,46 @@ func (tp *taskPlugin) Finished(success bool) error {
 		return err
 	}
 	defer file.Close()
+
+	tempFile, err := ioutil.TempFile(os.TempDir(), "livelog")
```

Comment:
Maybe use `runtime.Environment.TemporaryStorage` to get a temporary file in the temporary directory.

We mostly introduced the `TemporaryStorage` abstraction to ensure that files could be moved to something other than `/tmp`.. Notably because `/tmp` is a bad choice for docker images, cache volumes, etc...

So I'm not sure... if we should use it here.